docker/lnd/Dockerfile

```diff
@@ -34,3 +34,5 @@ RUN apk add --no-cache \
 # Copy the entrypoint script.
 COPY "docker/lnd/start-lnd.sh" .
 RUN chmod +x start-lnd.sh
+
+VOLUME /root/.lnd
```

Comment:
👍

Reply:
Looks like this isn't actually needed if we use a named volume in `docker-compose`